lib/sync.js

```diff
@@ -394,7 +403,7 @@ SyncApi.prototype._sync = function(syncOptions, attempt) {
         since: syncToken || undefined // do not send 'null'
     };
 
-    if (attempt > 1) {
+    if (self._syncConnectionLost) {
```

Comment:
ah right. If the objective is to emit the SYNCING event sooner, could make the comment say so?